src/main/java/de/uni/bremen/monty/moco/Main.java

```diff
@@ -243,10 +243,10 @@ public static void main(String[] args) throws IOException, InterruptedException
 		IOUtils.copy(Main.class.getResourceAsStream("/std_llvm_include.ll"), writer);
 
 		Package ast = buildPackage(inputFileName);
-		if (printAST) {
-			(new PrintVisitor()).visitDoubleDispatched(ast);
-			return;
-		}
+		// if (printAST) {
```

Comment:
indeed, I was not aware of that, although I always double-check the diff. I will fix that.